search-service/src/main/kotlin/com/egm/stellio/search/entity/web/EntityHandler.kt

```diff
@@ -203,11 +203,12 @@ class EntityHandler(
             contexts
         ).bind()
 
-        val csrFilters = CSRFilters(setOf(entityId))
+        val csrFilters =
+            CSRFilters(ids = setOf(entityId), operations = listOf(Operation.FEDERATION_OPS, Operation.RETRIEVE_ENTITY))
```

Comment:
you don't initialize `csf`, not sure something will happen in the service?

Reply:
it should be initialized with `csf=(operations==federationOps|operations==retrieveEntity`)` (even if when querying you only deal with `operations`)

Reply:
I choose to have both csf for getCSRs endpoint and operations for inside calls. Will you prefer to only have csf.

Reply:
cleaner to have only `csf`, no ? (calling from inside as if you were calling from outside)

Reply:
i thought and then i type "operations==federationOps|operations==retrieveEntity" and i changed my mind. you have no autocompletion and can't be sure your csf is valid.
A solution is to make a utils function csfBuilder that take a list of operation and build it for you.